src/components/Charts/Groupings.tsx

```diff
@@ -177,7 +177,7 @@ export const CHART_GROUPS: ChartGroup[] = [
     groupHeader: GroupHeader.ADDITIONAL_MISC,
     metricList: [
       {
-        metric: ExploreMetric.DEATHS,
+        metric: ExploreMetric.WEEKLY_DEATHS_PER_100K,
```

Comment:
So I don't think this is really "per 100k":
![image](https://user-images.githubusercontent.com/206364/163067423-672f85ab-0791-41d8-9840-4c6d80d8dcdd.png)

If California was really seeing 331 deaths per 100k, then over the course of a year (52 weeks), 17,212 people out of every 100,000 would be dead from covid, or 17% of the population of California.

So I think this should just be WEEKLY_DEATHS, not WEEKLY_DEATHS_PER_100K. 
 And we should remove the "per 100k" below.

Reply:
Wow, that's a lot of deaths! 😬 😆 I'll revise this.